containers/cpp-mariadb/.devcontainer/install-mariadb.sh

```diff
@@ -61,21 +61,50 @@ apt install -y libmariadb3 libmariadb-dev
 
 #Depending on the OS, install different C++ connectors
 find_os_props
-# Instructions are copied and modified from: https://mariadb.com/docs/clients/mariadb-connectors/connector-cpp/install/
-MARIADB_CONNECTOR=mariadb-connector-cpp-1.0.1-$OSURL
-cd ${TMP_DIR}
-curl -Ls https://dlm.mariadb.com/$OSTAG/connectors/cpp/connector-cpp-1.0.1/${MARIADB_CONNECTOR}.tar.gz -o ${MARIADB_CONNECTOR}.tar.gz
-tar -xvzf ${MARIADB_CONNECTOR}.tar.gz && cd ${MARIADB_CONNECTOR}
-install -d /usr/include/mariadb/conncpp
-
-#Header Files being copied into the necessary directories
-cp -R ./include/mariadb/* /usr/include/mariadb/
-cp -R ./include/mariadb/conncpp/* /usr/include/mariadb/conncpp
-cp -R ./include/mariadb/conncpp/compat/* /usr/include/mariadb/conncpp/compat
-
-install -d /usr/lib/mariadb
-install -d /usr/lib/mariadb/plugin
-
-#Shared libraries copied into usr/lib
-cp lib/mariadb/libmariadbcpp.so /usr/lib
-cp -R lib/mariadb/plugin/* /usr/lib/mariadb/plugin
+
+if [ "$(dpkg --print-architecture)" = "arm64" ] ; then
+    # Instructions are copied and modified from: https://github.com/mariadb-corporation/mariadb-connector-cpp/blob/master/BUILD.md
+    # and from: https://mariadb.com/docs/clients/mariadb-connectors/connector-cpp/install/
+    cd ${TMP_DIR}
+    apt-get update
+    apt-get install -y git cmake make gcc libssl-dev
```

Comment:
Are these tools only needed for building the connector? Does it makes sense to remove them after?

Reply:
well... the pattern that we saw in other scripts, include DCF, was to "apt install, and leave in place" ... will take another look though